packages/volto/locales/volto.pot

```diff
@@ -4869,6 +4961,11 @@ msgstr ""
 msgid "log in"
 msgstr ""
 
+#. Default: "login"
+#: helpers/MessageLabels/MessageLabels
+msgid "login"
```

Comment:
Just to clarify, "Log in" is the action of logging in (and is the previous translation message), whereas "Login" is usually the username. I'm not sure why this one was added, and I need more context to be clear.

```suggestion
#. Default: "Login"
#: helpers/MessageLabels/MessageLabels
msgid "Login"
```

Reply:
there are several msgids for `login`and `log in` so I would say that the originals are OK, I will capitalize the strings and see.